backend/spec/features/admin/products/edit/images_spec.rb

```diff
@@ -69,6 +69,18 @@
       expect(page).not_to have_field "image[alt]", with: "ruby on rails t-shirt"
     end
 
+    it "should show a no image placeholder when images fail to download" do
+      click_link "new_image_link"
+      within_fieldset 'New Image' do
+        attach_file('image_attachment', file_path)
+      end
+
+      click_button "Update"
+      Spree::Image.first.attachment.blob.update(key: 11)
```

Comment:
Specs are failing in the CI in jobs where we do not use Active Storage but Paperclip. In that case, `attachment` doesn't respond to `blob` and it fails. 

I think we need to find a way to make this compatible with both or skip the test when we are using Paperclip.

Reply:
Skipping test based on which module is used.